tests/jvm/src/test/scala/scala/meta/tests/semanticdb/OccurrenceSuite.scala

```diff
@@ -22,12 +22,8 @@ class OccurrenceSuite extends FunSuite {
 
   private def testBody(body: OccurrenceSuite.TestBody): Unit = {
     val expectedCompat =
-      if (ScalaVersion.atLeast212_14) body.expected
-      else
-        // Predef.type etc. was fixed in 2.12.14
-        body.expected.replace("Predef/*=>scala.Predef.*/.type", "Predef.type")
```

Comment:
is this unrelated? cleanup for 2.12?

Reply:
Yep, that's not necessary anymore and I was already cleaning this up.